FlowCrypt/Functionality/Services/Folders Services/RemoteFoldersProviderType/GmailService+folders.swift

```diff
@@ -62,7 +62,7 @@ private extension Folder {
 
         self.init(
             path: path,
-            name: name,
+            name: name.capitalized,
```

Comment:
I worry, this will have to be more complicated. Because if I have a custom label called `wHaTeVer` than I want it displayed exactly that way. But `INBOX` should be `Inbox`, in English.

The logic will have to be as follows:
 - if the label is a standard Gmail label (which you want to hard-code a list of them), then translate the label into local language. For now we only have English. That would be `Inbox`, `Sent`, `All Mail` and so on. The translations should be in the strings file.
 - for other labels, show them exactly as they are

List of standard labels:

<img width="174" alt="image" src="https://user-images.githubusercontent.com/6306961/159920672-c4f64783-0641-4ac9-af72-95a7a8800b47.png">

This may affect tests, which have to be updated to look for `Inbox` instead of `INBOX`, possibly.

Reply:
Nice idea. let me follow your suggestion